randovania/games/cave_story/logic_database/Sand Zone.txt

```diff
@@ -195,6 +195,7 @@ Extra - starting_script: <TRA0010:0099:0036:0033
                       Missile Launcher
                       Any of the following:
                           Missiles ≥ 50 and Kill Bosses with Missiles (Advanced)
+                          # Level 3 Missiles are mandatory, and all 3 need to hit for every shot
```

Comment:
I assume this is related to the recently opened issue, where we can't currently guarantee that someone has levelled up their weapon?